package.json

```diff
@@ -13,7 +13,20 @@
     "build": "gulp",
     "clean": "gulp clean",
     "clean-bin": "gulp clean-bin",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "copy:tokens": "cp -R dist/swift Tokens",
+    "cocoa:lint": "pod lib lint GravityParticles.podspec --allow-warnings",
+    "cocoa:package": "rm -r Tokens/ && npm run build && npm run copy:tokens",
+    "cocoa:prepare": "npm run cocoa:package && npm run cocoa:lint",
+    "cocoa:push": "pod trunk push GravityParticles.podspec",
+    "tag:version": "git tag $npm_package_version",
```

Comment:
Hmm... I'm concered that this scripts will conflict with stuff `semantic-release` will be doing for us. Also, `npm version` - which I think is the best way to update the package version in `package.json` will automatically do git tagging as well, so there's another potential clash there.

Reply:
yes as mentioned this will have overlaps with the #21,
resolving it as it was moved into the other branch and happy to discuss about it on how what route we chose.